Source/SimCacheModule/Source/Module/SimCacheModule.cpp

```diff
@@ -41,13 +44,20 @@ bool SimCacheModule::Initialize()
 		return false;
 	}
 
+	const bool InitializeCacheManagerSucceeded = InitializeCacheManager();
```

Comment:
(Should we put a `TODO` note here as a reminder re. our discussions on deferred loading?)

Reply:
I think we'll always initialize the cache manager here even with deferred loading. We'll probably have other actions that control when we load stuff based on the user opening the panel etc.

I'm not sure where I'd write that TODO though given that code doesn't exist yet 😛 . The work is tracked in mural & will be converted to tickets in the project so it won't be forgotten!

Reply:
Ah okay, that's understandable :P -- sounds good! :)